mlflow/utils/proto_json_utils.py

```diff
@@ -1,17 +1,79 @@
 import base64
 import datetime
 
+import json
 from json import JSONEncoder
 
 from google.protobuf.json_format import MessageToJson, ParseDict
+from google.protobuf.descriptor import FieldDescriptor
 
 from mlflow.exceptions import MlflowException
 from collections import defaultdict
+from functools import partial
+
+
+def _mark_int64_fields(proto_message):
+    """Converts a proto message to JSON, preserving only the int64/uint64/fixed64 fields."""
+
+    json_dict = {}
+    for field, value in proto_message.ListFields():
+        if field.type == FieldDescriptor.TYPE_MESSAGE:
+            ftype = partial(_mark_int64_fields)
+        elif field.type in [
+            FieldDescriptor.TYPE_INT64,
+            FieldDescriptor.TYPE_UINT64,
+            FieldDescriptor.TYPE_FIXED64,
```

Comment:
What about these types?

TYPE_FIXED32 = 7
TYPE_INT32 = 5
TYPE_SFIXED32 = 15
TYPE_SFIXED64 = 16
TYPE_SINT32 = 17
TYPE_SINT64 = 18
TYPE_UINT32 = 13
TYPE_UINT64 = 4

And CPP types:
CPPTYPE_INT32 = 1
CPPTYPE_INT64 = 2
CPPTYPE_UINT32 = 3
CPPTYPE_UINT64 = 4

https://googleapis.dev/python/protobuf/latest/google/protobuf/descriptor.html

Reply:
For int32 types, according to the [doc](https://developers.google.com/protocol-buffers/docs/proto3#json), they won't be converted to JSON strings so we don't need to add them here.

For CPP types, they are from a difference enum `FieldDescriptor::CppType` rather than `FieldDescriptor::Type`. They are used for `field.cpp_type()`. But here we only care about `field.type()`.
<img width="1896" alt="Screen Shot 2021-11-04 at 19 11 17" src="https://user-images.githubusercontent.com/5786126/140447798-a38def0b-26a0-4365-a3ae-c63779165a38.png">

I added two more int64 types (`TYPE_SFIXED64` and `TYPE_SINT64`) to cover all the int64 types.